src/page/Login.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types";
+
+import Input from "../component/form/Input";
+
+import './login.scss';
+
+const Login = ({handleSubmit, onUsernameChange, onPasswordChange, username, password, error}) => {
+
+    return (<>
+        <div className="login__background"></div>
+        <div className="login__card__frame">
+            <div className="login__card">
```

Comment:
The card can be defined as an article element.
```suggestion
            <article className="login__card">
```